packages/connectors/connector-qq/src/mock.ts

```diff
@@ -0,0 +1,40 @@
+export const mockedConfig = {
+  clientId: '<client-id>',
+  clientSecret: '<client-secret>',
+  redirectUri: 'http://localhost:3000/callback',
+};
+
+export const mockedAccessTokenResponse = {
+  access_token: 'access_token',
+  expires_in: '3600',
+  refresh_token: 'refresh_token',
+};
+
+export const mockedOpenIdAndUnionIdResponse = {
+  client_id: '<client-id>',
+  openid: 'openid',
+  unionid: 'unionid',
+};
+
+export const mockedUserInfoResponse = {
+  ret: 0,
+  msg: '',
+  is_lost: 0,
+  nickname: 'Ciallo',
+  gender: '男',
+  gender_type: 2,
+  province: '北京',
+  city: '海淀',
+  year: '1995',
+  figureurl: 'https://thirdqq.qlogo.cn/ek_qqapp/user-small.jpg',
+  figureurl_1: 'https://thirdqq.qlogo.cn/ek_qqapp/user-small.jpg',
+  figureurl_2: 'https://thirdqq.qlogo.cn/ek_qqapp/user-large.jpg',
+  figureurl_qq: 'https://thirdqq.qlogo.cn/ek_qqapp/user-qq-full.jpg',
+  figureurl_qq_1: 'https://thirdqq.qlogo.cn/ek_qqapp/user-small.jpg',
+  figureurl_qq_2: 'https://thirdqq.qlogo.cn/ek_qqapp/user-large.jpg',
```

Comment:
It would be better to use dummy data instead of meaningful informations in the mock response.